core/math/basis.h

```diff
@@ -146,7 +146,7 @@ class Basis {
 
 	bool is_equal_approx(const Basis &p_basis) const;
 	// TODO: Break compatibility in 4.0 by getting rid of this so that it's only an instance method. See also TODO in variant_call.cpp
-	bool is_equal_approx(const Basis &a, const Basis &b) const { return a.is_equal_approx(b); }
+	bool is_equal_approx(const Basis &a, const Basis &b) const { return is_equal_approx(a); }
```

Comment:
What is (*this) in `bool is_equal_approx(const Basis &a, const Basis &b) const { return is_equal_approx(a); }`

I wonder where b went.

Reply:
Well that was the bug, that's described in the OP. `*this` is the Basis instance you call this on. `a` is the Basis you compare to. `b` should not exist.